src/Rendering/babylon.utilityLayerRenderer.ts

```diff
@@ -59,6 +59,7 @@ module BABYLON {
 
                 let pointerEvent = <PointerEvent>(prePointerInfo.event);
                 if (originalScene!.isPointerCaptured(pointerEvent.pointerId)) {
+                    this._pointerCaptures[pointerEvent.pointerId] = false;

```

Comment:
Double check that adding this logic here is correct. It is needed because certain pointer up that were captured by the original scene left the utility layer pointer captured to true, resulting in the next pointer down on the utility layer to be ignored.